manager/integration/tests/test_scheduling.py

```diff
@@ -468,20 +467,30 @@ def test_replica_auto_balance_node_least_effort(client, volume_name):  # NOQA
     common.update_setting(client,
                           SETTING_REPLICA_AUTO_BALANCE, "least-effort")
 
-    n1, n2, n3 = client.list_node()
+    n_names = [get_self_host_id()]
+    nodes = client.list_node()
+    for node in nodes:
+        if node.name == n_names[0]:
+            continue
+        n_names.append(node.name)
+    assert len(n_names) == 3
+    assert n_names[0] == get_self_host_id()
+    n1 = client.by_id_node(n_names[0])
+    n2 = client.by_id_node(n_names[1])
+    n3 = client.by_id_node(n_names[2])
+
```

Comment:
Why we need this? It seems no different than before.

```python
n1, n2, n3 = client.list_node()
```

Reply:
`n1, n2, n3 = client.list_node()` This is the culprit of the unstable test result. This is because the case will write data from the current node, thus need to ensure n1 == get_self_host_id().